module/os/linux/zfs/zfs_vnops_os.c

```diff
@@ -3713,6 +3713,7 @@ zfs_inactive(struct inode *ip)
 		zfs_sa_upgrade_txholds(tx, zp);
 		error = dmu_tx_assign(tx, TXG_WAIT);
 		if (error) {
+			/* XXX we should at least log the error? */
```

Comment:
Since the TXG_WAIT is often done with a VERIFY0 I'd say don't worry about logging the error.

Reply:
Removed the XXX